.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "5.0"
+  - "5.1"
```

Comment:
ah, if this was for the build failure, this won't help :) Rest parameters were added in 6